integration/image/inspect_test.go

```diff
@@ -82,6 +82,76 @@ func TestImageInspectDescriptor(t *testing.T) {
 	assert.Check(t, inspect.Descriptor.Size > 0)
 }
 
+// Regression test for: https://github.com/moby/moby/issues/51566
+//
+// This can be reproduced with two image that share the same uncompressed layer
+// but have a different compressed blob is pulled.
+
```

Comment:
Probably should add `//` comment-lines for the empty lines so that it's all considered a single comment for the test;

```suggestion
//
```